tilings/strategies/experimental_verification.py

```diff
@@ -16,13 +16,93 @@
 from .abstract import BasisAwareVerificationStrategy
 
 __all__ = [
+    "NoRootCellVerificationStrategy",
     "ShortObstructionVerificationStrategy",
     "SubclassVerificationFactory",
 ]
 
 TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
 
 
+class NoRootCellVerificationStrategy(BasisAwareVerificationStrategy):
+    """
+    A strategy to mark as verified any tiling that does not contain the root
+    basis localized in a cell. Tilings with dimensions 1x1 are ignored.
+    """
+
+    def __init__(
+        self,
+        basis: Optional[Iterable[Perm]] = None,
+        symmetry: bool = False,
+        ignore_parent: bool = True,
+    ):
+        super().__init__(basis=basis, symmetry=symmetry, ignore_parent=ignore_parent)
```

Comment:
you can remove this since you just call super 
although you changed the default of ignore parent, if that was on purpose then ignore this